### ISSUE TITLE
Clap plugins

### ISSUE THREAD

**Issue by RustoMCSpit** [Maerorr/maerors-vst3-plugins#2]
As you're using nih-plug, can you export to clap?

**Comment by Maerorr** [comment]
Yes, nih-plug allows for clap exporting, you just need to change `nih_export_vst3!()` to `nih_export_clap!()` at the end of each lib.rs file and possibly fill out the clap id and descriptions for it to build.

**Comment by RustoMCSpit** [comment]
> Yes, nih-plug allows for clap exporting, you just need to change `nih_export_vst3!()` to `nih_export_clap!()` at the end of each lib.rs file and possibly fill out the clap id and descriptions for it to build.

could you build them and put it into releases? i want to add this to https://github.com/RustoMCSpit/awesome-linux-clap-list/blob/main/README.md
